plugins/catalog-backend/src/ingestion/processors/codeowners/resolve.ts

```diff
@@ -35,11 +39,11 @@ export function resolveCodeOwner(
 }
 
 export function normalizeCodeOwner(owner: string) {
-  if (owner.match(/^@.*\/.*/)) {
+  if (owner.match(GROUP_PATTERN)) {
     return owner.split('/')[1];
-  } else if (owner.match(/^@.*/)) {
-    return owner.substring(1);
-  } else if (owner.match(/^.*@.*\..*$/)) {
+  } else if (owner.match(USER_PATTERN)) {
+    return `user:${owner.substring(1)}`;
```

Comment:
```suggestion
    return `User:${owner.substring(1)}`;
```

It actually is case sensitive, but as a matter of taste, maybe let's keep the original casing.

Reply:
Thanks! I thought it looked funny alright.